src/Apps/ArtAdvisor/07-Curated-Discovery/server.ts

```diff
@@ -1,11 +1,31 @@
 import express from "express"
 import { ArtsyRequest, ArtsyResponse } from "Server/middleware/artsyExpress"
 import _ from "lodash"
+import { WeaviateDB } from "./weaviate-db"
+
+const weaviateDB = new WeaviateDB()
 
 const inferBudgetIntent = async (req: ArtsyRequest, res: ArtsyResponse) => {
   res.json({ amount: 42 })
 }
 
+const getMarketingCollections = async (
+  req: ArtsyRequest,
+  res: ArtsyResponse
+) => {
+  let { concepts, budget } = req.query
```

Comment:
While we accept the budge query param, we're not doing anything with it, yet. We will need to wire this up fully once we know exactly what kind of input to expect.